tools/cpp/BUILD.static

```diff
@@ -96,6 +97,20 @@ cc_toolchain(
     supports_param_files = 1,
 )
 
+cc_toolchain(
+    name = "cc-compiler-x86_32_windows",
```

Comment:
This rule too is a copy of the 64-bit one. Please do the same as I ask for in `tools/cpp/BUILD`: rename the 64-bit rule to indicate it's also for 32-bit, and use it for both architectures.